README.md

```diff
@@ -147,7 +147,7 @@ It supports backpressure.
 
 Publish the given packet to subscribed clients and functions. It supports backpressure.
 
-A packet must contain the following properties:
+A packet contains the following properties:
 
```

Comment:
Contradicting sentences. This one states that all properties are mandatory but the second says only topic (and as I suspect, the payload property) is mandatory.